### DISCUSSION TITLE
/posts/2023/04/20Vue3+TS(uniapp)%E6%89%8B%E6%92%B8%E4%B8%80%E4%B8%AA%E8%81%8A%E5%A4%A9%E9%A1%B5%E9%9D%A2

### DISCUSSION THREAD

**Discussion by giscus** [Justin3go/justin3go.com#62] [Announcements]
# /posts/2023/04/20Vue3+TS(uniapp)%E6%89%8B%E6%92%B8%E4%B8%80%E4%B8%AA%E8%81%8A%E5%A4%A9%E9%A1%B5%E9%9D%A2

坚持深耕技术领域的T型前端程序员, 关注独立开发与AI，喜欢Vuejs、Nestjs, 还会点Python、搜索引擎、NLP、Web3、后端

https://justin3go.com/posts/2023/04/20Vue3+TS(uniapp)%E6%89%8B%E6%92%B8%E4%B8%80%E4%B8%AA%E8%81%8A%E5%A4%A9%E9%A1%B5%E9%9D%A2

**Comment by xinlong-chen865** [comment]
最后的链表存储有必要吗？为什么不存在一个key中呢？

**Reply by Justin3go** [reply]
我当时应该是为了懒加载，如果消息过多，一下子加载所有消息不太合理；
而这种链表就是：每次向上滚动时，加载最近一天内的消息这样

**Reply by xinlong-chen865** [reply]
为什么用链表呢，而不是一个简单队列？

**Reply by Justin3go** [reply]
Localstorage是key/value存储的，如果用队列是不是只能一次性将所有数据取出来

**Reply by Justin3go** [reply]
不过大多数好像还是直接使用IndexDB存储的
